crates/proto/src/version/v662/enums/actor_type.rs

```diff
@@ -2,7 +2,7 @@ use bedrockrs_macros::ProtoCodec;
 use bitflags::bitflags;
 
 bitflags! {
-    struct Flags: isize {
```

Comment:
Is using `isize` correct here for packets?! I think packets have the same structure independent of the host computers arch..

Reply:
not sure what you're asking lol, but it should be i32 cause the enum is i32. otherwise I'd have to type cast every single one in the enum

Reply:
isize is the size of a signed pointer, aka an i32 on 32-bit systems and i64 on 64-bit systems. But we should use i32s directly

Reply:
yeah it is an i32